app/presenters/scope_filters_presenter.rb

```diff
@@ -1,29 +1,42 @@
 require 'gds_api/support_api'
 
 class ScopeFiltersPresenter
-  attr_reader :path, :organisation_slug
+  attr_reader :paths, :organisation_slug, :paths_for_api
 
-  def initialize(path: nil, organisation_slug: nil)
-    @path = normalize_path(path)
+  def initialize(paths: nil, organisation_slug: nil)
+    @paths = normalize_paths(paths)
     @organisation_slug = organisation_slug
   end
 
   def filtered?
-    path.present? || organisation_slug.present?
+    paths.present? || organisation_slug.present?
   end
 
   def invalid_filter?
     !filtered?
   end
 
   def done_page?
-    path.present? ? path.start_with?("done", "/done") : false
+    paths.present? ? paths.split(', ').any? { |path| path.start_with?("done", "/done") } : false
   end
 
   def organisation_title
     organisation["title"] if organisation.present?
   end
 
+  def paths_title
+    if paths.present?
+      parsed_paths = paths.split(',').map(&:strip)
```

Comment:
Oh! Maybe `parsed_paths` is a good internal name for the array version of `paths` / `paths_for_api`?

Reply:
:+1: used this